### PR TITLE
Fix Presto Hive Metastore JDBC URL

### PR DESCRIPTION
**Pull request by rimolive**

### CODE REVIEW

File: presto/base/presto-cr.yaml [hunk 1]
```diff
@@ -103,7 +103,7 @@ spec:
       image:
         imagepullPolicy: IfNotPresent
       postgreSql:
-        jdbcUrl: "postgresql://datacatalog:datacatalog@postgres:5432/datacatalog"
+        jdbcUrl: "jdbc:postgresql://postgres.dh-dev-presto.svc.cluster.local:5432/datacatalog"
```
Comment (accorvin) [inline presto/base/presto-cr.yaml]: It looks like this value is specific to a dev environment. Should it not go in the dev overlay?

**Review by accorvin — CHANGES_REQUESTED** [review]

File: presto/base/presto-cr.yaml (hunk 1 above)
Comment (rimolive) [reply presto/base/presto-cr.yaml]: Moved to dev overlay

**Comment by accorvin** [comment]
There is a way to use patches in overlays such that you don't need to duplicate the whole file. We should implement that here. I believe you can find an example here: https://github.com/AICoE/idh-manifests/blob/production/argo-events/overlays/dh-prod-argo/kustomization.yaml but perhaps @HumairAK can point you to better examples.

File: presto/overlays/dev/presto-cr.yaml [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+      type: COUNTER
+    enabled: true
+
+  service:
+    name: presto-service
+    type: ClusterIP
+
+
+  image:
+    name: starburstdata/presto:338-e.3-k8s-0.41
```
Comment (gmfrasca) [inline presto/overlays/dev/presto-cr.yaml]: please move to quay?

File: presto/overlays/dev/presto-cr.yaml [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+      cpu: 1
+      image:
+        imagepullPolicy: IfNotPresent
+      postgreSql:
+        jdbcUrl: "jdbc:postgresql://postgres.dh-dev-presto.svc.cluster.local:5432/datacatalog"
+        username: "datacatalog"
+        password: "datacatalog"
+      s3Endpoint: https://s3.upshift.redhat.com/
+      image:
+        name: starburstdata/hive-metastore:k8s-0.7.osc
```
Comment (gmfrasca) [inline presto/overlays/dev/presto-cr.yaml]: also probably needs to move to quay

File: presto/overlays/dev/presto-cr.yaml (hunk 2 above)
Comment (rimolive) [reply presto/overlays/dev/presto-cr.yaml]: Good point, but the idea behind this PR is to have a dev overlay with the PostgreSQL JDBC URL configured for that environment. I can work on another PR to fix this.

File: presto/overlays/dev/presto-cr.yaml (hunk 3 above)
Comment (rimolive) [reply presto/overlays/dev/presto-cr.yaml]: See my previous comment

**Review by gmfrasca — APPROVED** [review]
:pizza:

**Comment by gmfrasca** [comment]
looks like you need to rebase of `production`, fyi

**Review by accorvin — APPROVED** [review]
Looks good pending a rebase.

**Pull request merged**